clusterloader2/pkg/measurement/common/bundle/test_metrics.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bundle
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	testMetricsMeasurementName = "TestMetrics"
+)
+
+func init() {
+	measurement.Register(testMetricsMeasurementName, createTestMetricsMeasurment)
+}
+
+func createTestMetricsMeasurment() measurement.Measurement {
+	var metrics testMetrics
+	var err error
+	if metrics.apiResponsiveness, err = measurement.CreateMeasurement("APIResponsiveness"); err != nil {
```

Comment:
This can fail test, right? Then it shouldn't be here.

Reply:
APIResponsiveness removed from the bundle.